lib/reverse_sentence.rb

```diff
@@ -1,6 +1,54 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n^3)
```

Comment:
The time complexity is not O(n<sup>3</sup>).  

Instead you have:

O(n + n + m * w + m)

Since you will have relatively fewer words than letters (where n is the number of letters), and most words will be short, you could say this is `O(n)` time complexity.